tests/lib/hardware/libvirt.py

```diff
@@ -429,7 +429,12 @@ def boot_nodes(self, masters: int, workers: int, offset: int = 0):
         for t in threads:
             t.join()
 
-    def destroy(self):
-        super().destroy()
+    def destroy(self, skip=False):
+        super().destroy(skip=skip)
+
+        if skip:
+            logger.warning(f"Leaving network {self._network.name()}")
```

Comment:
There are also nodes and disks . Maybe mention these, too?

Reply:
They get mentioned from the super method on hardware_base.